src/test/scala/com/target/data_validator/validator/TestHelpersSpec.scala

```diff
@@ -0,0 +1,82 @@
+package com.target.data_validator.validator
+
+import com.target.TestingSparkSession
+import com.target.data_validator.TestHelpers._
```

Comment:
`import com.target.TestHelpers._`

Reply:
I get error with `import com.target.TestHelpers._`

Reply:
There are some more comments above that are being collapsed... Maybe you didn't see them?  I suggested renaming the package of `TestHelpers` to match the location of the file in the test directory.

> `package com.target` to correspond to location in directory

Reply:
Github was hiding them. I will address them and update pr.